var/spack/repos/builtin/packages/intel-tbb/package.py

```diff
@@ -238,3 +238,21 @@ def libs(self):
         shared = True if '+shared' in self.spec else False
         return find_libraries(
             'libtbb*', root=self.prefix, shared=shared, recursive=True)
+
+    @run_after('install')
+    def install_pkgconfig(self):
+        libdir = self.spec['tbb'].libs.directories[0]
+        pkg_path = join_path(libdir, 'pkgconfig')
+        mkdirp(pkg_path)
+
+        with open(join_path(pkg_path, 'tbb.pc'), 'w') as f:
+            f.write('prefix={0}\n'.format(self.prefix))
+            f.write('exec_prefix=${prefix}\n')
+            f.write('libdir={0}\n'.format(libdir))
+            f.write('includedir={0}\n'.format(self.prefix.include))
+            f.write('\n')
+            f.write('Name: Threading Building Blocks\n')
+            f.write('Description: Intel\'s parallelism library for C++\n')
+            f.write('Version: {0}\n'.format(self.spec.version))
+            f.write('Cflags: -I${includedir}\n')
+            f.write('Libs: -L${libdir} -ltbb -latomic\n')
```

Comment:
Seems `-latomic` comes from `pmemkv` (see https://github.com/oneapi-src/oneTBB/pull/361), but I have no clear understanding why it's needed.
@karczex, @alexey-katranov you may want to take a look.

Reply:
> I have no clear understanding why it's needed.

I don't understand it either ... , but `-latomic` is included in some pre-built packages. So added that.

https://github.com/oneapi-src/oneTBB/pull/361#discussion_r641711014

https://github.com/numba/numba/pull/2217#discussion_r91325175

Reply:
I found the source of `-latomic`. this seems to affect openvdb.

`-latomic` was introduced because of https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=787084.

https://sources.debian.org/src/tbb/2020.3-1/debian/changelog/?hl=293#L293

```
 [ Adam Conrad ]
  * gcc-atomic-detection.patch: Fix the gcc atomic detection to work for
    gcc-5.x and up, as well as adding -latomic to LIBS (closes: #787084)
  * gcc-atomic-always.patch: Always define TBB_USE_GCC_BUILTINS for GCC.
  * debian/tbb.pc.in: Add -latomic to the pkg-config file for the above.
 ```



patch:  https://salsa.debian.org/science-team/tbb/-/blob/master/debian/patches/gcc-atomic-detection.patch